### PR TITLE
Process review comments

### PR DESCRIPTION
**Pull request by vnbaaij**
# Pull Request

### General

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-blazor/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component

### CODE REVIEW

File: global.json
Comment (dvoituron) [inline global.json:1]: Why did you add this file ?
Is it compatible with multiple .NET targets ?

File: src/Microsoft.Fast.Components.FluentUI/Components/Progress/FluentProgress.razor [hunk 1]
```diff
@@ -1,12 +1,15 @@
 @namespace Microsoft.Fast.Components.FluentUI
 @inherits FluentComponentBase
-<fluent-progress @ref=Element
+@if (Visible)
+{
+    <fluent-progress @ref=Element
                  class="@ClassValue"
```
Comment (dvoituron) [inline src/Microsoft.Fast.Components.FluentUI/Components/Progress/FluentProgress.razor:6]: Could you align these attributes?

**Review by dvoituron — CHANGES_REQUESTED** [review]

File: src/Microsoft.Fast.Components.FluentUI/Components/InputFile/FluentInputFile.razor [hunk 2]
```diff
@@ -26,11 +26,10 @@
         </div>
     </div>
 
-    <div style="grid-column: 1; grid-row: 1; z-index: @(DropOver ? 999 : null)">
+    <div style="grid-column: 1; grid-row: 1; z-index: @(DropOver ? ZIndex.DropOver : null)">
```
[the remark this answers is not in the document]
Comment (vnbaaij) [reply src/Microsoft.Fast.Components.FluentUI/Components/InputFile/FluentInputFile.razor:29]: Changed to `InputFileDropZone`

**Review by dvoituron — APPROVED** [review]
